### DISCUSSION TITLE
best practice for naming page styles

### DISCUSSION THREAD

**Discussion by popuguytheparrot** [vercel/next.js#17912] [Help] [answered]
Hi.

How to be with styles for page. I mean 
about.module.scss
profile.module.scss

it be like component in upper first letter? About.module.scss
And how to be with code splitting?
```jsx

import React from 'react'
import styles from 'features/about/about.module.scss

export default function About(){...}
```

**Comment by jamesmosier** [comment, marked as answer]
Hi @popuguytheparrot. There isn't a right or wrong way to go about it! What I see most often though is all lowercase/kebab-case:

```
about.module.scss
my-page.module.scss
```

And as far as code splitting, Next optimizes both your JavaScript and CSS for production.

From the [docs](https://nextjs.org/docs/basic-features/built-in-css-support):

> In production, all CSS Module files will be automatically concatenated into many minified and code-split .css files
